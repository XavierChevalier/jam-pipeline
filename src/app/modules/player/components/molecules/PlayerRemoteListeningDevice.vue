<script setup lang="ts">
  import Icon from '@/app/components/atoms/AppIcon.vue'
  import { mdiMusicVideoOutline } from '@/assets/images/icons/MdiExtra'

  defineProps({
    remoteDeviceName: {
      type: String,
      required: true,
    },
  })
</script>

<template>
  <span class="text-primary">
    <Icon size="sm" :path="mdiMusicVideoOutline" />
    <span class="ml-1 align-middle">{{ remoteDeviceName }}</span>
  </span>
</template>
