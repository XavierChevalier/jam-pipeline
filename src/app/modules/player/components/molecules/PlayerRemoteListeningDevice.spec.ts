import * as Stories from './PlayerRemoteListeningDevice.stories'
import { StorybookTestsGenerator } from '@/tests/tests-generators/StorybookTestsGenerator'

describe('PlayerRemoteListeningDevice', () => {
  StorybookTestsGenerator.fromStoriesExports(Stories)
    .snapshotEachStories()
    .property<string>('remoteDeviceName', (property) => {
      property.itShouldBeDefined().itShouldBeRequired().itShouldBeTypeOf(String)
    })
})
