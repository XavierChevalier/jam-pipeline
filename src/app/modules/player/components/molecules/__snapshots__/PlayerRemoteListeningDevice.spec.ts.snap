// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerRemoteListeningDevice stories snapshots [snapshot] should render the "ListeningOnDesktop" component story correctly 1`] = `
<span
  class="text-primary"
>
  <anonymous-stub
    path="M 3 3 C 1.895 3 1 3.895 1 5 L 1 17 C 1 18.105 1.895 19 3 19 L 8 19 L 8 21 L 16 21 L 16 19 L 21 19 C 22.105 19 23 18.105 23 17 L 23 5 C 23 3.895 22.105 3 21 3 L 3 3 z M 3 5 L 21 5 L 21 17 L 3 17 L 3 5 z M 11 7 L 11 11 A 2 2 0 0 0 9 13 A 2 2 0 0 0 11 15 A 2 2 0 0 0 13 13 L 13 9 L 15 9 L 15 7 L 11 7 z"
    size="sm"
  />
  <span
    class="ml-1 align-middle"
  >
    Desktop-B9M8G7J
  </span>
</span>
`;
